src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -121,23 +121,31 @@ def setUp(self):
                                                       '..') + os.path.sep)
         self.prefix = build_paths['PREFIX']
         self.ompi_prefix = build_paths["OMPI_PREFIX"]
-        self.tmp = os.path.join(self.prefix, 'tmp')
         self.daos_test = os.path.join(self.prefix, 'bin', 'daos_test')
         self.orterun = os.path.join(self.ompi_prefix, "bin", "orterun")
         self.daosctl = os.path.join(self.prefix, 'bin', 'daosctl')
 
+        # set default shared dir for daos tests in case DAOS_TEST_SHARED_DIR$
+        # is not set, for RPM env and non-RPM env.$
```

Comment:
Not sure why there are trailing `$` at the end of these comment lines.

Reply:
It's a copy/paste issue we also noticed in some python test code changes: I normally do ":set list" in vi editor to make sure no space trailer, but forgot to unlist them before making copies. Will clear them here.

Reply:
done